torch/_inductor/codegen/cuda/cutlass_epilogue_gen.py

```diff
@@ -236,24 +278,31 @@ def __init__(self, accumulator_node_name: str):
         Args:
             accumulator_node_name (str): The name of the accumulator node which should contain
                                           the Matmul result before fusion according to the IR graph.
+            pre_fused_evt_args (Optional[str]): Optional arguments for a pre-fused EVT expression (typically addmm args).
         """
         self.accumulator_node_name: str = accumulator_node_name  #
         self.output: IndentedBuffer = IndentedBuffer(0)  # The output buffer for codegen
         self.var_counter: int = (
             0  # used to generate variable names, incremented for each new variable
         )
+        self.pre_fused_evt_args: Optional[str] = pre_fused_evt_args
         self.aliases: Dict[str, str] = dict()  # Aliases for subexpression functors
 
     @staticmethod
     def ir_to_evt_argument_string(
         template_output_node_name: str,
         epilogue_nodes: List[IRNode],
+        pre_fused_evt_args: Optional[str] = None,
     ) -> str:
         formatter = CutlassEVTEpilogueArgumentFormatter(
-            template_output_node_name,
+            template_output_node_name, pre_fused_evt_args
         )
-
-        with virtualized.V.set_ops_handler(formatter), patch.object(
+        result = pre_fused_evt_args
```

Comment:
I believe, it would be cleaner to move this line (using `pre_fused_evt_args`) after the following `if` statement. A bit confusing otherwise.